ui/src/components/agency/graphContainer.vue

```diff
@@ -32,9 +56,36 @@
       </div>
       <graph-bar :options="chartOptions" :series="series"></graph-bar>
     </div>
-    <div v-show="this.executorLog != null" class="executorLog">
-      <h4>O Executor não conseguiu dados para esses mês e ano pois:</h4>
-      <h4>{{ this.executorLog }}</h4>
+    <div
+      v-show="this.executorLog.cmd != null"
+      class="errorLog"
+    >
+      <p><b>Erro no comando: </b>{{ this.executorLog.cmd }}</p>
+      <p><b> Saída de erro: </b>{{ this.executorLog.err }}</p>
+      <b> Variáveis de ambiente (env): </b>
+
+      <ul style="list-style: none;">
+        <li>{{ this.executorLog.env[0] }}</li>
```

Comment:
A quantidade de variáveis de ambiente pode mudar. Isso tem que estar dentro de um for-loop.

Reply:
Eu tentei usar o :v-for (como já tinha usado antes), mas eu não conseguir colocar para funcionar. Ficou dando um erro que não consegui resolver ai apelei para essa forma.Nem a documentação ajudou. Mas posso ver e tentar mais para ver se consigo.

Reply:
Por favor, tenta mais um pouco :)

Não deveria ser complicado imprimir uma lista de strings. Se for, é um problema sério do vue.js.